part-4/model-building-components.md

```diff
@@ -564,15 +566,111 @@ The screen should look like in the screen shot below:
+
+{hint style="warning"}
+If an event should be repeteadly fired during the simulation (i.e., the **One time** property is set to false), and the condition depends on systems state (parameter or state variables calculated during simulation), the even will only be fired if this condition is met within the defined output interval and resolution! This may lead to unexpected behavior if the output resolution is too coarse. For example, if the condition is `Concentration > 1 && Concentration < 2` and the concentration is 0.5 at time point t1 and 2 at time point t2, but no output is defined between t1 and t2, the event will not be fired since the condition was not met at any of the output time points.
+
+A discussion on this topic can be found in the [MoBi® Forum](https://github.com/Open-Systems-Pharmacology/MoBi/issues/321).
+{% endhint %}
+
+#### Events assignments
+
+An **assignment** defines what should be changed when the event is executed.‌ Each event can have multiple assignments. Each assignment has the following properties:
 
-An event is used to change an entity, like the amount of molecules or a reaction rate, when a given condition is met. This condition can be, for example, that a given simulation time is reached, or that the concentration of a molecule has exceeded a certain value. Thus, such a programmed event is used to reflect external changes to the simulation, like the application of a drug or a sudden physical change in the spatial structure, like a vessel rupture.
+- **Name**: A unique name for the assignment.
+- **Changed entity path**: Path to the parameter or molecule that should be changed by the event.
+- **Assignment**: New value or formula that should be assigned to the entity.
+- **Use assignment as value**: If true, the assignment is used as a calculated value at the time point of event execution. If false, the assignment (formula) overwrites the value (or other formula).
 
-In the drug delivery case, however, you rather want to use an application instead of an event to have more options available for drug release and repeated applications. Since the generation of an application in MoBi® can be rather complicated and is beyond the scope of this manual, we will restrict the description to adapting applications that were previously imported from PK-Sim®, where complex applications schemes can be generated more easily.
+### Container
 
-Events and applications are grouped in an events building block, where events are sub-grouped in event groups. To create such a structure, you may need to create a new events building block using the context menu of the building block explorer or the corresponding ribbon button of the Modeling & Simulation tab. In our example, an empty events building block named "Events" has already been created automatically. Simulations imported from a PK-Sim® project also contain such a building block that contains the applications.
+Containers can be created within:
```

Comment:
So basically, an "Event group" is nothing more than a logical container, which can contain further containers? I am confused by all the different levels of container types here.

Reply:
In principle yes.